src/com/ableton/PythonPackageVersionReader.groovy

```diff
@@ -0,0 +1,56 @@
+package com.ableton
+
+
+/**
+ * Class to get the version of a Python package by parsing the file that sets it.
+ */
+class PythonPackageVersionReader implements Serializable {
+  /**
+   * Script context.
+   * <strong>Required value, may not be null!</strong>
+   */
+  @SuppressWarnings('FieldTypeRequired')
+  def script = null
+
+  /**
+   * Start of the line setting the Python package version.
+   */
+  String versionLineBeginning = "__version__ = '"
+
+  /**
+   * End of the line setting the Python package version.
+   */
+  String versionLineEnding = "'"
+
+  /**
+   * Read a Python file setting the version of a package (usually {@code __init__.py})
+   * and return its version as a {@code String}.
+   * @param filename The file to read and parse the version out of.
+                     <strong>Required value, may not be null!</strong>
```

Comment:
I don't think this is necessary to document for parameters. Unless a default value is provided, it's usually assumed that they are required. We only assert parameters as a sanity check.

Reply:
Fixed as of 2437eb2.